### DISCUSSION TITLE
Noop transition animation

### DISCUSSION THREAD

**Discussion by Akxe** [withastro/roadmap#628] [Proposal]
### Body

# Summary

Ability to easily skip animation to/from a page.
I want to have an animation to a project page from the main page, but I do not want to show the reverse. 

# Background & Motivation

Often there is a title displayed over an image when the image is hovered. It makes sense to animate the text when the text is visible, but animating from the item page back, the title should not animate since the title is not visible.

# Goals

Add an other predefined animation keyword.

# Example

```html
<header transition:animate="noop">
<header transition:animate={{
  forwards: 'fade',
  backwards: 'noop',
}}>
```

**Comment by matthewp** [comment]
Can you post feedback on transitions API in the RFC? https://github.com/withastro/roadmap/pull/607

**Reply by Akxe** [reply]
done with some more hints
